crashreports/internal/unix/CrashReportContext.cpp

```diff
@@ -0,0 +1,138 @@
+
```

Comment:
Mostly for this file and its header, I omit remarks I would make analogous to Windows case.  There are a few others to make.